tollway/constants.py

```diff
@@ -8,3 +10,13 @@
 INCLUDE_LATE_RATE = 20
 INCLUDE_DUPLICATE_RATE = 50
 ALL_EVENTS_COUNT = 250
+
+
+class Help(Enum):
```

Comment:
uppercase for constant?

Reply:
yes, was thinking do uppercase since being used as a constant